File: sb_payment_sdk/src/main/java/com/startbutton/sb_payment_sdk/models/TransferDetailModel.java
package com.startbutton.sb_payment_sdk.models;

import com.google.gson.annotations.SerializedName;

public class TransferDetailModel {
    @SerializedName("accountNumber")
    private long accountNumber;

    @SerializedName("reference")
    private String reference;

    @SerializedName("expiryTime")
    private String expiryTime;

    @SerializedName("bankName")
    private String bankName;

    @SerializedName("accountName")
    private String accountName;

    public String getAccountNumber() {
        return String.valueOf(accountNumber);
    }

    public String getReference() {
        return reference;
    }

    public String getExpiryTime() {
        return expiryTime;
    }

    public String getBankName() {
        return bankName;
    }

    public String getAccountName() {
        return accountName;
    }
}

File: lib/models/transfer_detail_model.dart
class TransferDetailModel {
  final accountNumber;
  final reference;
  final expiryTime;
  final bankName;
  final accountName;

  TransferDetailModel(
      {this.accountNumber,
      this.reference,
      this.expiryTime,
      this.bankName,
      this.accountName});

  factory TransferDetailModel.fromJson(Map<String, dynamic> data){
    return TransferDetailModel(
        accountNumber: data["accountNumber"],
        reference: data["reference"],
        expiryTime: data["expiryTime"],
        bankName: data["bankName"],
        accountName: data["accountName"],
    );
  }
}